centreon/tests/api/behat.yml

```diff
@@ -18,12 +18,6 @@ default:
       paths: [ "%paths.base%/features/Clapi.feature" ]
       contexts:
         - Centreon\Test\Api\Context\ClapiContext
```

Comment:
Are we sure those tests have been replaced yet ?

Reply:
This will come in another PR by the QA